pkgs/development/python-modules/craft-platforms/default.nix

```diff
@@ -47,6 +47,10 @@ buildPythonPackage rec {
 
   pytestFlagsArray = [ "tests/unit" ];
 
+  disabledTests = [
+    "test_fuzz_get_platforms_build_plan_single_base"
```

Comment:
Usually a comment is added explaining why the test is disabled. To facilitate next reviews.

Reply:
Fixed :)

Reply:
Thank you!